packages/braid-design-system/src/lib/components/Disclosure/Disclosure.screenshots.tsx

```diff
@@ -1,58 +1,187 @@
-import React, { useState } from 'react';
+import type { ReactNode } from 'react';
+import React from 'react';
 import type { ComponentScreenshot } from 'site/types';
 import { Disclosure, Text } from '../';
+import { Box } from '../Box/Box';
+
+const Container = ({ children }: { children: ReactNode }) => (
+  <Box style={{ maxWidth: 250 }}>{children}</Box>
+);
```

Comment:
Was this intentional? The [screenshots on large breakpoints][wide] are now clamped at 250px, but the 320px ones [are also clamped][320].

[wide]: https://www.chromatic.com/component?appId=5bd0cd3e0cc977002443f476&csfId=disclosure--apac&buildNumber=4944&k=6405b97df693085eb45086ec-320-interactive-true&h=17&b=-4
[320]: https://www.chromatic.com/component?appId=5bd0cd3e0cc977002443f476&csfId=disclosure--apac&buildNumber=4944&k=6405b97df693085eb45086ec-320-snapshot-true&h=18&b=-5

Reply:
They are [only screenshot at 320] (see [screenshots]) and was done to deliberately exacerbate wrapping. At this size previously the chevron would wrap separately to the label, which was the bug that was fixed too.

[only screenshot at 320]: https://github.com/seek-oss/braid-design-system/pull/1240/files/e318dd33ad7831c668d2bc6fb3cf7770eabd7986#diff-e703586717ac85ca074180a6023ffdfa8e999e54cecfe4ca46a0f0141fda9a66R12
[screenshots]: https://www.chromatic.com/test?appId=5bd0cd3e0cc977002443f476&id=640579ae8bdb80c75963cd7b